Rehlate/index.html

```diff
@@ -0,0 +1,373 @@
+            </div>
+            <p>
+              كانت تجربة مميزة مع خدمات رائعة قضيت أفضل إجازة لي من فترة طويلة،
+              أنصح بالتعامل مع شركة رحلاتي بدون تردد.
+            </p>
+          </div>
+          <div class="review-card card-bg-grey-bg">
+            <img src="./assets/SVG/Stars.svg" alt="review" />
+            <div class="inline">
+            <img src="./assets/SVG/reviews/review-02.svg" alt="review-02" />
+            <h5>ماهر عبد اللطيف</h5>
+            </div>
+            <p>
+              تقدم شركة رحلاتي خدمات مناسبة جداً لرجال الأعمال، هذا ما ساعدني في
+              رحلتي وجعلني اختصر جداً من مشاكل الحجوزات.
+            </p>
+          </div>
+          <div class="review-card card-bg-grey-bg">
+            <img src="./assets/SVG/Stars.svg" alt="review" />
+            <div class="inline">
+            <img src="./assets/SVG/reviews/review-03.svg" alt="review-03" />
+            <h5>أسامة الأبيض</h5>
+            </div>
+            <p>
+              يمكنني القول إنني قمت بقطاء أفضل رحلة عائلة على الإطلاق، كان
+              الفنادق رائعة وخطة السفر ممتازة جداً ومريحة.
+            </p>
+          </div>
+        </div>
+        <div class="desktop">
+        <button>
+          <img src="./assets/navigators/arrow-left-purple.svg" />
+        </button>
+        <button >
+          <img src="./assets/navigators/arrow-right-purple.svg" />
+        </button>
+        </div>
+      </section>
+
+      <section class="contact">
+        <h4 class="heading-grey">هل لديك سؤال؟</h4>
+        <h2 class="black">يسعدنا التواصل معك</h2>
+        <div>
+          <div class="contact-form">
+            <form class="pink-bg">
+                <span>
+              <label>الاسم</label>
+              <input type="text" placeholder="يرجى تسجيل اسمك الكامل" />
+                </span>
+            <span>
+              <label>البريد الإلكتروني</label>
+              <input type="email" placeholder="سجل بريدك الشخصي" />
+            </span>
+            <span>
+              <label>الموضوع</label>
+              <textarea
+                placeholder="ما هو السؤال أو المشكلة التي لديك؟"
+              ></textarea>
+            </span>
+              <button>أرسل الرسالة</button>
+            </form>
+          </div>
+          <ul>
+            <li>
+              <img
+                src="./assets/SVG/contact/help-icon.svg"
+                alt="contact-icon-01"
+              />
+              <div class="form-inline">
+              <h5 class="black">نقدم لك المساعدة</h5>
+              <p class="card-grey">
+                إن كان لديك اي مشكلة في البلد الذي تسافر إليه سنقوم بمساعدك في
+                هذا الموضوع.
+              </p>
+              </div>
+            </li>
+            <li>
+              <img
+                src="./assets/SVG/contact/communication-icon.svg"
+                alt="contact-icon-02"
+              />
+              <div class="form-inline">
+              <h5 class="black">للتواصل</h5>
```

Comment:
please increase the space between the` <h5>` and `<p>` in this section (in mobile version only)